cmd/bridge/main.go

```diff
@@ -260,17 +254,17 @@ func main() {
 		var err error
 
 		if srv.DexClient, err = auth.NewDexClient(*fDexAPIHost, *fDexClientCAFile, *fDexClientCertFile, *fDexClientKeyFile); err != nil {
-			log.Fatalf("Failed to create a Dex API client: %v", err)
+			klog.Fatalf("Failed to create a Dex API client: %v", err)
 		}
 	}
 
 	var secureCookies bool
 	if baseURL.Scheme == "https" {
 		secureCookies = true
-		log.Info("cookies are secure!")
+		klog.V(4).Info("cookies are secure!")
 	} else {
 		secureCookies = false
-		log.Warning("cookies are not secure because base-address is not https!")
+		klog.V(4).Info("cookies are not secure because base-address is not https!")
```

Comment:
Lets display this by default, as we are doing now
```suggestion
		klog.Info("cookies are not secure because base-address is not https!")
```